fs/build

```diff
@@ -157,6 +175,7 @@ mkdir -p $CADMIUMROOT/tmp/root/etc/sway
 cp $CADMIUMROOT/board/$TARGET/sway-config $CADMIUMROOT/tmp/root/root/swayconfig
 
 # copy over ucm files
+mkdir -p $CADMIUMROOT/tmp/root/usr/share/alsa/ucm2/
```

Comment:
if you need this then arch likely has this in other place

Reply:
I didn't fiddle with sound.